### PR TITLE
[12.0][MIG] account_skip_bank_reconciliation

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

File: account_skip_bank_reconciliation/models/account_bank_statement.py [hunk 1]
```diff
@@ -1,19 +1,19 @@
-# © 20118 Eficent Business and IT Consulting Services S.L. (www.eficent.com)
+# © 2018-19 Eficent Business and IT Consulting Services S.L. (www.eficent.com)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 from odoo import models
+from odoo.osv import expression
 
 
 class AccountBankStatementLine(models.Model):
     _inherit = "account.bank.statement.line"
```
Comment (CasVissers-360ERP) [inline account_skip_bank_reconciliation/models/account_bank_statement.py]: Wrong inherit here, should be:
class AccountReconciliation(models.AbstractModel):
    _inherit = 'account.reconciliation.widget'

After this the module works great!

**Review by CasVissers-360ERP — CHANGES_REQUESTED** [review]

File: account_skip_bank_reconciliation/models/account_bank_statement.py (hunk 1 above)
Comment (MiquelRForgeFlow) [reply account_skip_bank_reconciliation/models/account_bank_statement.py]: ops, oversight. fixed, thanks :+1:

**Comment by CasVissers-360ERP** [comment]
@mreficent Maybe we should also change the filename? Sorry I forgot to say that in first comment.

**Comment by MiquelRForgeFlow** [comment]
done 😉

**Review by CasVissers-360ERP — APPROVED** [review]
LGTM! Thnx!

**Review by astirpe — APPROVED** [review]

**Comment by CasVissers-360ERP** [comment]
@mreficent I think we miss an inherit here. In the filtering the accounts are filtered. But the automatic reconciliation activated when a partner is set, does still search the skip accounts. I tried to find the method myself but didn't succeed (yet).

It might be _domain_move_lines_for_manual_reconciliation and/or _get_move_line_reconciliation_proposition both methods don't have a domain for exclude_bank_reconcile.

**Comment by MiquelRForgeFlow** [comment]
@CasVissers can you check?

**Review by AdriaGForgeFlow — APPROVED** [review]
Functional test :+1:

**Comment by CasVissers-360ERP** [comment]
@MiquelRForgeFlow 
Sorry for the delay but it took me some time for figure out how to reproduce the usecase that didnt work. I was finally able to do it on the runbot related to this PR:
https://drive.google.com/file/d/1-zTmi7xfreF2KlSGNI648q8164wgXcZh/view

**Comment by JordiBForgeFlow** [comment]
@CasVissers you are right! I just fixed that. Can you please check again? It looks good from my end

File: account_skip_bank_reconciliation/models/account_reconcile_model.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# © 2018-19 Eficent Business and IT Consulting Services S.L. (www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class AccountReconcileModel(models.AbstractModel):
```
Comment (MiquelRForgeFlow) [inline account_skip_bank_reconciliation/models/account_reconcile_model.py]:
```suggestion
class AccountReconcileModel(models.Model):
```

Comment (JordiBForgeFlow) [reply account_skip_bank_reconciliation/models/account_reconcile_model.py]: @MiquelRForgeFlow fixed!

**Review by CasVissers-360ERP — APPROVED** [review]
NIce! My last issue is fixed. Again thanks for the great module.

**Pull request merged**